CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 - fixed saving and loading the music timestamp when the load current music option is enabled (#1237)
 - fixed the remember played music option always being enabled (#1249, regression since 2.16)
 - fixed the underwater SFX playing for one frame at the start of Palace Midas (#1251)
+- fixed Lara saying "No" when saving or loading near a puzzle hole or keyhole (#1255)
```

Comment:
We'll need changelog entries for the other issues as well.

Reply:
I combined the saying "no" issue since they stemmed from the same issue. No changelog for #1254 because it's unreleased.

Reply:
We still should mention #1268 in the brackets.